prusti-tests/tests/cargo_verify/veribetrfs/src/base/mod.rs

```diff
@@ -0,0 +1,9 @@
+pub mod option;
+pub mod vector;
+
+pub use option::Opt;
+pub use vector::Vector;
+
+pub fn replace<T>(dest: &mut T, src: T) -> T {
```

Comment:
You can use the `std::mem::replace` directly by giving it specs as shown [here](https://github.com/viperproject/prusti-dev/blob/e51e0c21b5f0ec5ad7e7ca9b3b8eda322965c0cb/prusti-tests/tests/verify/pass/extern-spec/swap.rs).